src/lake/Lake/Build/Actions.lean

```diff
@@ -39,7 +45,18 @@ def compileLeanModule (name : String) (leanFile : FilePath)
     ]
   }
 
-def compileO (name : String) (oFile srcFile : FilePath)
+def compileBcToO (name : String) (oFile srcFile : FilePath)
```

Comment:
I mentioned to this to Siddharth before in a DM, so it may already be on his TODO list. However, it seems worth repeating here. As far as I can tell, it seems unnecessary to have both `BcToO` and `CToO` variants of `compileO` (and the rename of `buildO`). Instead, `buildLeanBcToO` could just pass addition arguments to `compileO` and use clang as its `compiler`.